src/snapred/backend/dao/ingredients/ArtificialNormalizationIngredients.py

```diff
@@ -0,0 +1,12 @@
+from pydantic import BaseModel
+
+
+class ArtificialNormalizationIngredients(BaseModel):
+    """
+    Class to hold ingredients for the creationon of artificial normalization data.
+    """
+
+    peakWindowClippingSize: int = 10
```

Comment:
If this doesn't get modified elsewhere I would remove this and set it in Config. If you do need this here, I would still set the value with something defined in Config

Reply:
Ah, so the reason I left this in this way since I will be taking a user input for this value but I can still write the default within application.yml.